src/mapnode.cpp

```diff
@@ -564,7 +564,7 @@ u8 MapNode::getLevel(const NodeDefManager *nodemgr) const
 	// todo: after update in all games leave only if (f.param_type_2 ==
 	if(f.liquid_type == LIQUID_SOURCE)
 		return LIQUID_LEVEL_SOURCE;
-	if (f.param_type_2 == CPT2_FLOWINGLIQUID)
+	if (f.param_type_2 == CPT2_FLOWINGLIQUID || f.param_type_2 == CPT2_COLORED_FLOWINGLIQUID)
```

Comment:
Same here: Shouldn't `liquid_type` be consistent with `param_type_2`? Can we remove the below `if` as the comment suggests?